app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h3>Add Ingredient</h3>
+
+<%= form_with model: @recipe_food, url: recipe_recipe_foods_path(@recipe) do |f| %>
+  <div>
+    <%= f.label :food_id, "Food" %>
+    <%= f.collection_select :food_id, @foods, :id, :name, prompt: "Select a food", class: 'form-control' %>
+  </div>
+  <br>
+  <div>
+    <%= f.label :quantity %>
+    <%= f.number_field :quantity, autofocus: true, autocomplete: "quantity", class: 'form-control' %>
+  </div>
+  <br>
+  <div>
+    <%= f.submit "Add Ingredient", class: 'btn btn-outline-dark btn-sm' %>
```

Comment:
- [ ] Good job on this feature 👏 , however, i tried to click the cancel button in an attempt that i changed my mind to add an ingredient, but the button does not work. Kindly look at the path and fix this bug to have a seamless webapp 👍 

### View:
![image](https://github.com/ritikarawat220/Recipe-App/assets/50990925/1f955667-41f7-49b6-a7ea-52998a9508ae)